<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PixelReiter</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="settingsMenu" >
    <property name="title" >
     <string>&amp;Settings</string>
    </property>
    <widget class="QMenu" name="menuGrid" >
     <property name="title" >
      <string>Show &amp;Grid</string>
     </property>
     <addaction name="actionGridNone" />
     <addaction name="actionGridWhite" />
     <addaction name="actionGridBlack" />
    </widget>
    <addaction name="menuGrid" />
   </widget>
   <widget class="QMenu" name="editMenu" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="copyAct" />
   </widget>
   <widget class="QMenu" name="fileMenu" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="separator" />
    <addaction name="grabAct" />
    <addaction name="separator" />
    <addaction name="importScreenshotAct" />
    <addaction name="exportCurrentAct" />
    <addaction name="separator" />
    <addaction name="quitAct" />
   </widget>
   <addaction name="fileMenu" />
   <addaction name="editMenu" />
   <addaction name="settingsMenu" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="fileToolBar" >
   <property name="windowTitle" >
    <string>File</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="grabAct" />
   <addaction name="separator" />
   <addaction name="actionZoomIn" />
   <addaction name="actionZoomOut" />
  </widget>
  <widget class="QDockWidget" name="colorsDock" >
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea|Qt::NoDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Colors</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>9</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QTreeView" name="colorsTable" />
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>151</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="copyColorButton" >
         <property name="text" >
          <string>&amp;copy</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolsToolbar" >
   <property name="windowTitle" >
    <string>Tools</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>1</number>
   </attribute>
   <addaction name="actionColorPicker" />
   <addaction name="actionDraw" />
  </widget>
  <action name="grabAct" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Grab</string>
   </property>
   <property name="iconText" >
    <string>&amp;Grab</string>
   </property>
   <property name="statusTip" >
    <string>Update Screenshot (Space)</string>
   </property>
   <property name="shortcut" >
    <string>Space</string>
   </property>
  </action>
  <action name="quitAct" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/tools/icons/exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip" >
    <string>Quit Application (Ctrl+Q)</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="exportCurrentAct" >
   <property name="text" >
    <string>&amp;Export...</string>
   </property>
   <property name="statusTip" >
    <string>export current view</string>
   </property>
  </action>
  <action name="importScreenshotAct" >
   <property name="text" >
    <string>&amp;Import...</string>
   </property>
   <property name="statusTip" >
    <string>import screenshot</string>
   </property>
  </action>
  <action name="actionGridNone" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>none</string>
   </property>
  </action>
  <action name="actionGridWhite" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>white</string>
   </property>
  </action>
  <action name="actionGridBlack" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>black</string>
   </property>
  </action>
  <action name="actionZoomIn" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/tools/icons/viewmag+.png</iconset>
   </property>
   <property name="text" >
    <string>+</string>
   </property>
   <property name="toolTip" >
    <string>zoom in</string>
   </property>
   <property name="shortcut" >
    <string>+</string>
   </property>
  </action>
  <action name="actionZoomOut" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/tools/icons/viewmag-.png</iconset>
   </property>
   <property name="text" >
    <string>-</string>
   </property>
   <property name="shortcut" >
    <string>-</string>
   </property>
  </action>
  <action name="copyAct" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/tools/icons/editcopy.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
   <property name="statusTip" >
    <string>Copy to Clipboard (Ctrl+C)</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionColorPicker" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="icons.qrc" >:/tools/icons/tool_colorpicker.png</iconset>
   </property>
   <property name="text" >
    <string>Color Picker</string>
   </property>
  </action>
  <action name="actionDraw" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="icons.qrc" >:/tools/icons/paintbrush.png</iconset>
   </property>
   <property name="text" >
    <string>Draw</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections/>
</ui>
